packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -128,6 +128,15 @@ export class ProjectService {
             user.organizationUuid,
             data,
         );
+
+        // Give admin user permissions to user who created this project even if he is an admin
+        if (user.email) {
```

Comment:
what is the case when the `user.email` does not exist?

Reply:
The user type has email as optional but it shouldn't happen. It was marked optional since when we query the user from the DB the email comes from a different table so technically it could not find a match in that table. But I think we should make the property required and throw an error if an email is not found in the DB.

Reply:
ok thanks